_search-plugins/neural-sparse-search.md

```diff
@@ -261,6 +262,10 @@ GET my-nlp-index/_search
   }
 }
 ```
+## Step 5: Create and enable two-phase processor (Optional)
+'neural_sparse_two_phase_processor' is a new feature which introduced in OpenSearch 2.15. It can speed up the neural sparse query's time cost with negligible accurency loss
+For more information, you can refer to [neural-sparse-query-two-phase-processor]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/neural-sparse-query-two-phase-processor/).
+
```

Comment:
Can we give some example API calling like the section `## Setting a default model on an index or field`? Using all default values is also good, this would help users a lot

Reply:
We should also mention that this processor is strongly recommended to set

Reply:
Gave a example API for set a default 2-phase pipeline.  
Add a explanation for why we recommend this pipeline.